extensions/applicationinsights-clickanalytics-js/src/applicationinsights-clickanalytics-js.ts

```diff
@@ -2,6 +2,6 @@
 // Licensed under the MIT License.
 
 import { BehaviorEnumValidator, BehaviorMapValidator, BehaviorValueValidator, ClickAnalyticsPlugin } from "./ClickAnalyticsPlugin";
-import { IClickAnalyticsConfiguration } from "./Interfaces/Datamodel";
+import { IClickAnalyticsConfiguration, IValueCallback, ICustomDataTags } from "./Interfaces/Datamodel";
```

Comment:
Also add 
- ICoreData
- IPageTags
- IPageActionTelemetry
- ICustomProperties

and any other referenced interfaces or enums from these interfaces